test/stack.go

```diff
@@ -19,19 +19,18 @@ import (
 
 	"github.com/madlambda/spells/assert"
 	"github.com/mineiros-io/terramate/hcl"
-	"github.com/mineiros-io/terramate/stack"
 )
 
 // AssertStackImports checks that the given stack has all the wanted import
 // definitions. The wanted imports is a slice of the sources that are imported
-// on each block.
-func AssertStackImports(t *testing.T, rootdir string, got stack.S, want []string) {
+// on each block. The got stack must be not nil.
```

Comment:
```suggestion
// on each block.
```

The stack instance was refactored away